core/requirements/core/REQ_job-success.adoc

```diff
@@ -34,7 +34,7 @@ The value of the `type` parameter SHALL indicate the type of server that created
 
 [.component,class=part]
 ---
-The value of the `type` parameter SHALL be `process` to indicate that a server that conforms to this Standard created the job.
+The value of the `type` parameter SHALL be `ogc-api-processes` to indicate that a server that conforms to this Standard created the job.
```

Comment:
`type` not replaced by `processingEntityType`

Should the `profileEntityType` be mentioned here as well?